src/App.vue

```diff
@@ -32,27 +32,32 @@
       }
     },
     async created() {
+      //First try AEX-2
```

Comment:
why do we need to first try AEX-2 if we laready have a wallet? (eg. foundWallet is true?)

Reply:
`foundWallet` if false when `created` is called

Reply:
I know we had issues with the app being re-rendered in the past but I assume we can attend to them, when they appear again.